README.md

```diff
@@ -151,3 +151,7 @@ Below you can find information on how to push local code changes to a kind clust
 - push image to local registry: `podman push localhost:5000/maiqueb/multus-dynamic-networks-controller`
 - update manifests to use the generated image: `IMAGE_REGISTRY=localhost:5000/maiqueb make manifests`
 - deploy the controller: `kubectl apply -f manifests/dynamic-networks-controller.yaml`
+
+## Known limitations
```

Comment:
Could we mention device plugin based CNIs? And that requested network resources must be available on the node, we won't reschedule if they are not

Reply:
Sure. 

But I do not plan on supporting that for the near future.